packages/nodejs/test/credentialDeployment.test.ts

```diff
@@ -4,11 +4,11 @@ import {
     CredentialDeploymentTransaction,
     AttributeKey,
     IdentityInput,
+    createCredentialDeploymentTransaction,
+    TransactionExpiry,
+    getCredentialDeploymentSignDigest,
 } from '@concordium/common-sdk';
-import { createCredentialDeploymentTransaction } from '@concordium/common-sdk';
-import { TransactionExpiry } from '@concordium/common-sdk';
 import * as ed from 'noble-ed25519';
```

Comment:
Shouldn't the linter have found this? And all the other places in our tests with the same old import.